src/GlobalStyle.js

```diff
@@ -4,13 +4,8 @@ const { normalize } = require('styled-normalize');
 const GlobalStyle = createGlobalStyle`
   ${normalize}
 
-  html {
-    font-size: 62.5%
```

Comment:
What's the rationale to remove this? How does adoption of material UI affect the font-sizing?  🤔

Reply:
This affected the fonts in the Material UI components, e.g. making the button texts look small. Probably better to apply styling on smaller groups rather than using global style for font sizes.